webrtc.html

```diff
@@ -3780,6 +3780,7 @@
     readonly        attribute DOMString?              sdpMid;
     readonly        attribute unsigned short?         sdpMLineIndex;
     readonly        attribute DOMString?              foundation;
+    readonly        attribute byte?                   component;
```

Comment:
I forgot, `byte` is signed ([−128, 127]) while `octet` isn't, so I guess we want `octet` here.